### PR TITLE
[t134622] [MIFR15] [DEV] Migration v15 - sale-workflow (15.0.project_MI_465)

### PR DESCRIPTION
**Pull request by BT-atornero**
<div> Links to Odoo: </div> <ul>
<li><a target="_blank" href="https://braintec.com/web#view_type=form&model=project.task&id=134622">[t134622] [MIFR15] [DEV] Migration v15 - sale-workflow (15.0.project_MI_465)</a></li>
</ul>
<div>Affected Modules:</div>
<table><thead><tr><th>Module</th><th>Ext</th></tr></thead>
<tr><td>sale_order_lot_selection</td><td>.py</td></tr>
<tr><td>sale_order_product_recommendation</td><td>.py</td></tr>
<tr><td>sale_advance_payment</td><td>.py</td></tr>
</tbody></table>

### CODE REVIEW

File: sale_sourced_by_line/tests/test_sale_sourced_by_line.py [hunk 1]
```diff
@@ -23,6 +23,25 @@ def setUp(self):
             {"name": "Test Warehouse", "code": "TWH"}
         )
 
+        # # TODO: Two tests failed because of a product not having a route set.
```
Comment (BT-dherreros) [inline sale_sourced_by_line/tests/test_sale_sourced_by_line.py]: There is no point on keeping commented code, does this come from the OCA?

**Review by BT-dherreros — CHANGES_REQUESTED** [review]
Are all the changes we have on our side over OCA related to unit tests?

File: sale_sourced_by_line/tests/test_sale_sourced_by_line.py (hunk 1 above)
Comment (BT-atornero) [reply sale_sourced_by_line/tests/test_sale_sourced_by_line.py]: no, it is from braintec

**Comment by BT-atornero** [comment]
> Are all the changes we have on our side over OCA related to unit tests?

there is also a change in a wizard in sale_order_product_recommendation module

**Comment by BT-atornero** [comment]
I checked it a second time and found no further changes

**Review by BT-dherreros — APPROVED** [review]

**Pull request merged**